styles/globals.scss

```diff
@@ -82,6 +83,48 @@ body {
     }
   }
 
+  .header-icon {
+    margin: 0 2rem;
+    text-align: center;
+
+    [class^='ca-gov-icon-'] {
+      color: #116c97;
+      display: block;
+      font-size: 2rem;
+      margin: 0.5rem;
+    }
+
+    @media (min-width: 1200px) {
```

Comment:
I assume these breakpoints match the UIO website. Is that right?

Reply:
Yes, though I did not re-check for this ticket. Tina & I confirmed our breakpoints in #210